README.md

```diff
@@ -35,7 +35,7 @@ Install & Upgrade Tests |
 [![RefApp 2.x Patient Encounter](https://github.com/openmrs/openmrs-contrib-qaframework/actions/workflows/refapp-2x-patient-encounter.yml/badge.svg)](https://github.com/openmrs/openmrs-contrib-qaframework/actions/workflows/refapp-2x-patient-encounter.yml)
 [![RefApp 2.x Report](https://github.com/openmrs/openmrs-contrib-qaframework/actions/workflows/refapp-2x-report.yml/badge.svg)](https://github.com/openmrs/openmrs-contrib-qaframework/actions/workflows/refapp-2x-report.yml)
 [![RefApp 2.x Phone Number Validation](https://github.com/openmrs/openmrs-contrib-qaframework/actions/workflows/refapp-2x-phoneNumberValidation.yml/badge.svg)](https://github.com/openmrs/openmrs-contrib-qaframework/actions/workflows/refapp-2x-phoneNumberValidation.yml)
-[![RefApp 2.x Provider](https://github.com/openmrs/openmrs-contrib-qaframework/actions/workflows/refapp-2x-provider.yml/badge.svg)](https://github.com/openmrs/openmrs-contrib-qaframework/actions/workflows/refapp-2x-provider.yml)
+[![RefApp 2.x Provider](https://github.com/openmrs/openmrs-contrib-qaframework/actions/workflows/refapp-2x-provider.yml/badge.svg?branch=master)](https://github.com/openmrs/openmrs-contrib-qaframework/actions/workflows/refapp-2x-provider.yml)
```

Comment:
You do not need to add `branch=master` because it's a default value. 
We use it to take the action result badge for another branch instead of the same branch.